File: src/main/java/nz/co/searchwellington/commentfeeds/detectors/CommentSlashFeedDetector.java
package nz.co.searchwellington.commentfeeds.detectors;

import org.springframework.stereotype.Component;

import java.net.URL;

@Component
public class CommentSlashFeedDetector implements CommentFeedDetector {

    @Override
    public boolean isValid(URL url) {
        return new GenericCommentFeedDetector("^.*\\/comments/feed\\/$").isValid(url);
    }
}

File: src/main/scala/nz/co/searchwellington/commentfeeds/detectors/CommentSlashFeedDetector.scala
package nz.co.searchwellington.commentfeeds.detectors

import org.springframework.stereotype.Component
import java.net.URL

@Component
class CommentSlashFeedDetector extends CommentFeedDetector {
  override def isValid(url: URL) = new GenericCommentFeedDetector("^.*\\/comments/feed\\/$").isValid(url)
}